Formula/libtool.rb

```diff
@@ -39,5 +39,12 @@ def caveats; <<-EOS.undent
 
   test do
     system "#{bin}/glibtool", "execute", "/usr/bin/true"
+    (testpath/"hello.c").write <<-EOS
+      #include <stdio.h>
+      int main() { puts("Hello, world!"); return 0; }
+    EOS
+    system *%W[#{bin}/glibtool --mode=compile --tag=CC #{ENV.cc} -c hello.c -o hello.o]
```

Comment:
Not seeing a particular upside to use this format instead of the usual `system bin/"glibtool", "blah", "blah"` format, really, especially since this runs past the 80 character mark.